src/memex/__init__.py

```diff
@@ -3,6 +3,12 @@
 __all__ = ('__version__',)
 __version__ = '0.39.0+dev'
 
+GROUPFINDER_KEY = 'memex.groupfinder'
+
+
+def set_groupfinder(config, func):
+    config.registry[GROUPFINDER_KEY] = func
```

Comment:
Looks good, but as a matter of convention Pyramid configuration directives should usually accept "dotted-string" arguments as it makes it much easier to avoid import cycles. There's a `maybe_dotted` method on the configurator instance to help with this.